src/bin/pgcopydb/extensions.c

```diff
@@ -200,31 +200,71 @@ copydb_copy_extensions_hook(void *ctx, SourceExtension *ext)
 		{
 			SourceExtensionConfig *config = &(ext->config.array[i]);
 
-			log_info("COPY extension \"%s\" "
-					 "configuration table \"%s\".\"%s\"",
-					 ext->extname,
-					 config->nspname,
-					 config->relname);
-
 			char qname[PG_NAMEDATALEN_FQ] = { 0 };
 
 			sformat(qname, sizeof(qname), "%s.%s",
 					config->nspname,
 					config->relname);
 
-			CopyArgs args = {
-				.srcQname = qname,
-				.srcAttrList = "*",
-				.srcWhereClause = config->condition,
-				.dstQname = qname,
-				.dstAttrList = "",
-				.bytesTransmitted = 0
-			};
-
-			if (!pg_copy(src, dst, &args))
+			/*
+			 * According to the PostgreSQL documentation, relkind 'r' is a
+			 * regular table, and 'S' is a sequence.
+			 * https://www.postgresql.org/docs/current/catalog-pg-class.html
+			 */
+			if (config->relkind == 'r')
+			{
```

Comment:
Nitpicking: can we use `switch (config->relkind)` here? And if then the code in the case parts is too large, maybe refactor it in its own function?

Reply:
Done :)